src/main/java/edu/utdallas/davisbase/representation/SelectCommandRepresentation.java

```diff
@@ -0,0 +1,38 @@
+package edu.utdallas.davisbase.representation;
+
+import java.util.List;
+
+public class SelectCommandRepresentation implements CommandRepresentation {
+
+  private String command;
+  private List<String> columns;
+  private String table;
+  private boolean all;
+
+  public SelectCommandRepresentation(String command, String table, List<String> columns, boolean all){
+    this.command=command;
+    this.columns = columns;
```

Comment:
```suggestion
    this.columns = Collections.unmodifiableList(new ArrayList<>(columns));
```

(compile-time defensive programming + run-time fail-fast immutability)